src/queue.c

```diff
@@ -0,0 +1,101 @@
+#include "queue.h"
+
+#include <stdio.h>
+#include <stdlib.h>
+#define TRUE 1
+#define FALSE 0
+
+struct node
+{
+    long start_time;
+    long wait_time;
+    struct node *next;
+};
+typedef struct node node;
+
+struct queue
+{
+    int count;
+    node *front;
+    node *rear;
+};
+typedef struct queue queue;
+
+void initialize(queue *q)
+{
+    q->count = 0;
+    q->front = NULL;
+    q->rear = NULL;
+}
+
+void display(node *head)
```

Comment:
Prefix these functions with your object. This displays a node. So make it `node_display`. This is to prevent C from calling some other `display` function in some other library.